docs/configuration.rst

```diff
@@ -325,6 +349,50 @@ Removes a variant set from a given data repository and dataset.
 
     $ ga4gh_repo remove-variantset path/to/datarepo aDataset aVariantSet
 
++++++++++++++++
+add-biosample
++++++++++++++++
+
+Add a BioSample record to the given data repository and dataset. Records
+should be described in JSON format according to the GA4GH protocol. To 
+associate this BioSample with an Individual record add the optional 
+individualName flag.
```

Comment:
This uniqueness is required by the server's architecture, and is not meant to reflect the schemas. This name is also the filename of the record with the extension removed.